### PR TITLE
Docs: clean up image renderer dependencies and add ones for CentOS 8

### LINKED ISSUE #32799: Docs feedback: /administration/image_rendering.md
In the **Grafana image renderer plugin and remote rendering service** section, there is a list of dependencies to install on a CentOS system. This was up to date on CentOS 7.

On CentOS 8, the following two dependencies are also required: `alsa-lib` and `libX11-xcb`. The `libX11-xcb` package doesn't seem to exist in CentOS 7 repositories so it looks like we need 2 lists now.

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What this PR does / why we need it**:
Clean up some duplicates in the image renderer dependencies (thanks @kobtea for finding them) and add ones for CentOS 8.

**Which issue(s) this PR fixes**:
Fixes https://github.com/grafana/grafana/issues/32799

**Special notes for your reviewer**:
Based on https://github.com/grafana/grafana/pull/39014

### CODE REVIEW

**Comment by marefr** [comment]
@AgnesToulet think this should be backported to v8.3.x now instead perhaps? Docs for v8.2.x are no longer published since the release of v8.3.0.

**Comment by AgnesToulet** [comment]
@adeverteuil @achatterjee-grafana Would you have some time to review this PR please?

File: docs/sources/image-rendering/troubleshooting.md [hunk 1]
```diff
@@ -58,31 +58,37 @@ ldd chrome-linux/chrome
...
 **Centos:**
 
-On a minimal Centos installation, the following dependencies have been confirmed as needed for the image rendering to function:
+On a minimal CentOS 7 installation, the following dependencies have been confirmed as needed for the image rendering to function:
 
 ```bash
 libXcomposite libXdamage libXtst cups libXScrnSaver pango atk adwaita-cursor-theme adwaita-icon-theme at at-spi2-atk at-spi2-core cairo-gobject colord-libs dconf desktop-file-utils ed emacs-filesystem gdk-pixbuf2 glib-networking gnutls gsettings-desktop-schemas gtk-update-icon-cache gtk3 hicolor-icon-theme jasper-libs json-glib libappindicator-gtk3 libdbusmenu libdbusmenu-gtk3 libepoxy liberation-fonts liberation-narrow-fonts liberation-sans-fonts liberation-serif-fonts libgusb libindicator-gtk3 libmodman libproxy libsoup libwayland-cursor libwayland-egl libxkbcommon m4 mailx nettle patch psmisc redhat-lsb-core redhat-lsb-submod-security rest spax time trousers xdg-utils xkeyboard-config alsa-lib
 ```
 
+On a minimal CentOS 8 installation, the following dependencies have been confirmed as needed for the image rendering to function:
```
Comment (achatterjee-grafana) [inline docs/sources/image-rendering/troubleshooting.md:86]:
```suggestion
On a minimal CentOS 8 installation, the following dependencies are required for the image rendering to function:
```

**Review by achatterjee-grafana — APPROVED** [review]
Requested a minor change.

File: docs/sources/image-rendering/troubleshooting.md (hunk 1 above)
Comment (adeverteuil) [reply docs/sources/image-rendering/troubleshooting.md:86]: Yes, this reads better.

**Pull request merged**